outdated-pkgs

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/python
+
+"""
+Usage:
+  For goc packages:
+      %s goc      [3.3|3.4]     [el6|el7] [development|testing|release] [itb|production]
+  For osg epel packages:
+      %s osg-epel [3.4|3.5|3.5] [el6|el7] [development|testing|release]
+    Or:
+      %s osg-epel [devops]      [el7|el8] [itb|production]
+  For osg i2 packages:
+      %s osg-i2   [3.3|3.4]     [el6|el7] [development|testing|release]
+
+Default for goc:
+  %s el%s %s %s
+Default for osg-epel:
+  %s el%s %s
+Default for osg-i2:
+  %s el%s %s
+
+Options:
+  --make-cache     save (goc-elX-YY.html, osg-3.X-elX-ZZ.list) | ( (epelX.html | i2_X.html) and osg-3.X-elX-XXX.list) for offline use
+  --cached         used cached files instead of pulling live versions
+  --[no-]color     colorize version differences (default = True if tty)
+  --html           generate html output (implies --color)
+  --preamble=text  print some preamble text before the report
+  --show-all       show versions even for non-outdated packages
+"""
+import sys
+import os
+import re
+import rpm
+import urllib2
+try:
+    from rpmUtils.miscutils import stringToVersion
+except ImportError:
+    from miscutils import stringToVersion
+
+MODE     = ''
+SERIES   = '3.5'
+EL       = '7' # default for i2 is 6
+TAG      = 'release'
+OSG_REPO = 'release'
+GOC_REPO = 'production'
+
+use_cached = False
+make_cache = False
+use_color  = sys.stdout.isatty()
+show_all   = False
+html_out   = False
+preamble   = None
+
+
+def usage(func):
+    # check which funcion user used, print different values
+    script = os.path.basename(__file__)
+    print __doc__ % (script, script, script, script,
+                    SERIES, EL, OSG_REPO, GOC_REPO,
+                    SERIES, EL, TAG,
+                    SERIES, EL, TAG)
+    sys.exit()
+
+# Mode selection
+if sys.argv[1] in ['goc', 'osg-epel', 'osg-i2']:
+    MODE = sys.argv[1]
+else:
+    usage()
+
+# Insert code below to change any default values
+if MODE == 'osg-i2':
+    # Since default EL is 6 for i2 pkgs
+    EL = '6'
+if MODE == 'goc':
+    SERIES = '3.4'
+
+for arg in sys.argv[2:]:
+    if MODE == 'goc':
+        if re.search(r'^el[5-9]$', arg):
+            EL = arg[2]
+        elif re.search(r'^[3-9]\.[0-9]$', arg):
+            SERIES = arg
+        elif re.search(r'^(upcoming-)?(development|testing|release)$', arg):
+            OSG_REPO = arg
+    elif MODE == 'osg-epel':
+        if re.search(r'^el[5-9]$', arg):
+            EL = arg[2]
+        elif re.search(r'^[3-9]\.[0-9]$', arg) or arg == 'devops':
+            SERIES = arg
+        elif re.search(r'^(upcoming-)?(development|testing|release)$', arg) \
+            or arg in ('itb', 'production'):
+            TAG = arg
+    elif MODE == 'osg-i2':
+        if re.search(r'^el[5-9]$', arg):
+            EL = arg[2]
+        elif re.search(r'^[3-9]\.[0-9]$', arg):
+            SERIES = arg
+        elif re.search(r'^(upcoming-)?(development|testing|release)$', arg):
+            TAG = arg
```

Comment:
```suggestion
    if re.search(r'^el[5-9]$', arg):
        EL = arg[2]
    elif re.search(r'^[3-9]\.[0-9]$', arg) or arg == 'devops':
        SERIES = arg
    elif re.search(r'^(upcoming-)?(development|testing|release)$', arg) \ 
        or arg in ('itb', 'production'):
        TAG = arg
```

Seems like all this can be collapsed, and then you can change existing `OSG_REPO` references to `TAG`